web/libs/editor/src/components/HtxTextBox/HtxTextBox.jsx

```diff
@@ -178,12 +182,32 @@ export class HtxTextBox extends React.Component {
       <>
         <Paragraph {...props}>
           <span ref={this.textRef}>{text}</span>
-          {isEditable && onChange && (
-            <EditOutlined onClick={this.startEditing} aria-label="Edit Region" className="ant-typography-edit" />
-          )}
         </Paragraph>
+        {isEditable && onChange && (
+        <Tooltip title="Edit">
+          <Elem
+            tag={Button}
+            name="edit"
+            type="text"
+            className="button__icon edit"
```

Comment:
when using sass modules for styling you can reference the child class sort of how @bmartel  and I did here https://github.com/HumanSignal/label-studio/blob/d3c75f2bc2e6e1e9317142f1c202cdfefd3fbddd/web/libs/ui/src/lib/checkbox/checkbox.tsx#L31 
in this case we're replacing a mod but so we have some conditionals but if wanted it to always apply it as you would in this case we would have done something like `className={styles.checkbox_checked}`